src/main/resources/hudson/plugins/sauce_ondemand/credentials/SauceCredentials/credentials.jelly

```diff
@@ -3,9 +3,8 @@
     <f:entry title="${%Username}" field="username">
         <f:textbox/>
     </f:entry>
-    <f:entry title="${%APIKey}" field="apiKey">
+    <f:entry title="${%Access Key}" field="apiKey">
         <f:textbox/>
     </f:entry>
-    <f:validateButton title="${%Test Connection}" with="username,apiKey" method="validate"/>
```

Comment:
now that the button is gone, can you delete the associated function as well?
(I think its called doValidate)